src/sagemaker/tensorflow/README.rst

```diff
@@ -19,6 +30,27 @@ Suppose that you already have a TensorFlow training script called
 ``tf-train.py``. You can train this script in SageMaker Learner as
 follows:
```

Comment:
following the colon with section headers seems a bit weird to me. I think I'd remove this paragraph entirely